pynamodb/migration.py

```diff
@@ -31,6 +32,9 @@ def _build_lba_filter_condition(attribute_names):
 def migrate_boolean_attributes(model_class,
                                attribute_names,
                                read_capacity_to_consume_per_second=10,
+                               page_size=None,
+                               limit=None,
+                               back_off=1,
```

Comment:
nit: if we've already published a release with `migrate_boolean_attributes` defined then move the new kwargs to the end